### ISSUE TITLE
build_sphinx is deprecated

### ISSUE THREAD

**Issue by ThomasWaldmann** [python-llfuse/python-llfuse#88]
```
running build_sphinx
/opt/hostedtoolcache/Python/3.11.3/x64/lib/python3.11/site-packages/setuptools/_distutils/dist.py:987: RemovedInSphinx70Warning: setup.py build_sphinx is deprecated.
  cmd_obj.run()
Running Sphinx v6.2.1
```

**Comment by ThomasWaldmann** [comment]
they removed the setuptools integration completely.
one now needs to just separately call `sphinx-build -b html rst doc/html`.

**Comment by ThomasWaldmann** [comment]
fixed by #87
